lucida/questionanswering/OpenEphyra/src/info/ephyra/nlp/StanfordNeTagger.java

```diff
@@ -7,11 +7,10 @@
 
 import edu.stanford.nlp.ie.AbstractSequenceClassifier;
 import edu.stanford.nlp.ie.crf.CRFClassifier;
-import info.ephyra.util.StringUtils;
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileOutputStream;
-import java.io.PrintWriter;
+// import java.io.File;
+// import java.io.FileNotFoundException;
+// import java.io.FileOutputStream;
+// import java.io.PrintWriter;

```

Comment:
Those modules are used by a part of `extractNEs` function which is commented out. It's probably some debugging stuff. We can remove these lines as well as PrintWriter code from `extractNEs` function